05-regression.Rmd

```diff
@@ -582,10 +582,10 @@ Let's inspect the individual columns and match them with the elements of Figure
 
 Just as we did for the instructor of the 21st course in the `evals_ch5` dataset (in the first row of the table), let's repeat the calculations for the instructor of the 24th course (in the fourth row of Table \@ref(tab:regression-points-1)):
 
-* `score` = 4.4 is the observed teaching `score` $y$ for this course's instructor.
-* `bty_avg` = 5.50 is the value of the explanatory variable `bty_avg` $x$ for this course's instructor.
-* `score_hat` = 4.25 = `r evals_line[1]` + `r evals_line[2]` $\cdot$ 5.50 is the fitted value $\widehat{y}$ on the regression line for this course's instructor.
-* `residual` = 0.153 =  4.4 - 4.25 is the value of the residual for this instructor. In other words, the model's fitted value was off by 0.153 teaching score units for this course's instructor.
+* `score` = `r regression_points$score[24]` is the observed teaching `score` $y$ for this course's instructor.
```

Comment:
Another good example. Let's compute, save, and comment these numerical values to variables in the R code chunk titled `regression-points-1`. 

This will do two things:
1. Make the code below more readable
1. Lessen the computation time